bin/compile

```diff
@@ -35,16 +38,38 @@ cp -n $BP_DIR/scripts/Procfile $BUILD_DIR/Procfile
 echo "-----> Copying static.json to application slug"
 cp -n $BP_DIR/scripts/static.json $BUILD_DIR/static.json
 
-echo "-----> Installing Bower package manager"
 cd $BUILD_DIR
-npm install bower
+if [ -d "./bower_components" ]; then
```

Comment:
That was my initial thought but my concern is that someone already relying on this buildpack will be lost if it changes out from under them. So, this way it keeps working but enables the optimization.

Of course, they'll never realize the option exists if things keep working... thoughts?